src/components/modal/Snackbar/index.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { FC, useEffect, useState } from 'react';
+
+import {
+  snackbarWrapperStyle,
+  snackbarHideStyle,
+  snackbarShowStyle,
+} from './style';
+
+type SnackbarProps = {
+  isOpen?: boolean;
+  message?: string;
+  color?: string;
+  backgroundColor?: string;
+};
```

Comment:
[저번 PR](https://github.com/telbby/telbby-frontend/pull/25#discussion_r754756664)에 `Context` 란 얘기를 해주시면서 이를 분리해주셨던 것 같은데, 이번에도 같은 이유로 분리를 해주신 건가요?

음... `SnackbarProps는 "SnackbarStateType"의 부분집합을 Props로 갖는다` 라는 문맥이 어색할까요?

```suggestion
type SnackbarProps = Partial<SnackbarStateType>;
```

Reply:
으음,,,,, Snackbar 컴포넌트는 글로벌 상태와는 별개라고 생각해서 상태와 관련된 의존성을 가지는 것은 별로 좋아보이지는 않습니다. 하지만 반대로 SnackbarState가 Snackbar 컴포넌트에 의존하는 것은 별로 어색해보이지 않아보이네요.

말씀하신 의존 관계의 반대로 리팩토링하는 것은 어떤가용? SnackbarStateType을 SnackbarProps로 정의하는 방식이랄까요.

Reply:
좋습니다!! 덕분에 모듈 의존 관계에 대해서 고민해보며 생각할 수 있게 되었네요 :) 
감사합니다~

Reply:
ㅎㅎ,,,, 사실 지금 지금 방식이 맞는 지는 잘 모르겠습니다만 나중에 더 나은 방식, 구조를 알게 되면 바로 리팩토링 하도록 하겠습니다

Reply:
넵넵... 혹시 더 좋은 방식이 생각나게 되면 공유 부탁드립니다 :)

Reply:
수정했습니다.

관련 커밋: [e7ba00b](https://github.com/telbby/telbby-frontend/pull/28/commits/e7ba00b93059458d8ecfefb5069d8da368161e07), [9484f60](https://github.com/telbby/telbby-frontend/pull/28/commits/9484f60544d53b55102a90d0db691358cfd8b38b)